com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -69,6 +70,8 @@ public class Bob {
     public static final String VARIANT_RELEASE = "release";
     public static final String VARIANT_HEADLESS = "headless";
 
+    public static final String ARTIFACTS_URL = "http://d.defold.com/archive/";
```

Comment:
I think it would make sense to make this configurable, either through a command line option, or by checking for the presence of an environment variable. We do check for `DM_ARCHIVE_DOMAIN` in `build.py`. Perhaps we could. check for something similar, and fall back to `d.defold.com` if none is set. It could ofc be done in a separate PR or at a later time.

Reply:
If you don't mind, I keep it as it is now.